arm-advisor/2017-04-19/swagger/advisor.json

```diff
@@ -0,0 +1,625 @@
+				"description": "Enables the activation of a snoozed or dismissed recommendation. The snoozed or dismissed attribute of a recommendation is referred to as a suppression.",
+				"operationId": "Suppressions_Delete",
+				"parameters": [{
+						"name": "resourceUri",
+						"in": "path",
+						"description": "The fully qualified Azure Resource Manager identifier of the resource to which the recommendation applies.",
+						"required": true,
+						"type": "string"
+					}, {
+						"name": "recommendationId",
+						"in": "path",
+						"description": "The recommendation ID.",
+						"required": true,
+						"type": "string"
+					}, {
+						"name": "name",
+						"in": "path",
+						"description": "The name of the suppression.",
+						"required": true,
+						"type": "string"
+					}, {
+						"$ref": "#/parameters/apiVersionParameter"
+					}
+				],
+				"responses": {
+					"204": {
+						"description": "NoContent. The recommendation has been activated."
+					}
+				},
+				"deprecated": false,
+				"x-ms-examples": {
+					"DeleteSuppression": {
+						"$ref": "../examples/DeleteSuppression.json"
+					}
+				}
+			}
+		},
+		"/{resourceUri}/providers/Microsoft.Advisor/recommendations/{recommendationId}": {
+			"get": {
+				"tags": ["GetRecommendations"],
+				"description": "Obtains details of a cached recommendation.",
+				"operationId": "Recommendations_Get",
+				"parameters": [{
+						"name": "resourceUri",
+						"in": "path",
+						"description": "The fully qualified Azure Resource Manager identifier of the resource to which the recommendation applies.",
+						"required": true,
+						"type": "string"
+					}, {
+						"name": "recommendationId",
+						"in": "path",
+						"description": "The recommendation ID.",
+						"required": true,
+						"type": "string"
+					}, {
+						"$ref": "#/parameters/apiVersionParameter"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK. Successfully got recommendation detail.",
+						"schema": {
+							"$ref": "#/definitions/ResourceRecommendationBase"
+						}
+					}
+				},
+				"deprecated": false,
+				"x-ms-examples": {
+					"GetRecommendationDetail": {
+						"$ref": "../examples/GetRecommendationDetail.json"
+					}
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/suppressions": {
+			"get": {
+				"tags": ["Suppressions"],
+				"description": "Retrieves the list of snoozed or dismissed suppressions for a subscription. The snoozed or dismissed attribute of a recommendation is referred to as a suppression.",
+				"operationId": "Suppressions_List",
+				"parameters": [{
+						"$ref": "#/parameters/subscriptionIdParameter"
+					}, {
+						"$ref": "#/parameters/apiVersionParameter"
+					}, {
+						"name": "$top",
+						"in": "query",
+						"description": "The number of suppressions per page if a paged version of this API is being used.",
+						"required": false,
+						"type": "integer"
```

Comment:
You should also add ` "format": "int32"`. That way typed languages can efficiently use the memory for this variable.